setup.py

```diff
@@ -21,7 +21,7 @@ def readme():
       license='MIT',
       packages=['n50'],
       install_requires=[
-          'biopython',
+          'biopython','pandas'
```

Comment:
When I try to run the tests I get an error that matplotlib is missing, this will need to be added as a dependency